NetCore/project.json

```diff
@@ -5,18 +5,19 @@
     "emitEntryPoint": true
   },
   "frameworks": {
-    "netcoreapp1.0": {
+    "netcoreapp1.1": {
```

Comment:
Yeah, I assumed you did.

My library runs on netcoreapp1.0, and I was able to run my test from within your app on netcoreapp1.0, but when I try to run the full benchmark, BenchmarkDotNet's generated app hangs and then crashes.  When I have a strange problem like that with another library, my first troubleshooting step is to update to the latest version.  Unfortunately, their latest version requires netcoreapp 1.1...

This is my first experience with BenchmarkDotNet, so I have no clue how to diagnose what's causing the problem, but the update fixes it.  You may have better luck in your environment, but I didn't want to make a pull request for a config that doesn't work on my machine, and I didn't want you to exclude my imaging library from your tests just because the chosen benchmark platform has a problem with it.

Do you have any suggestions?